shaptools/hana.py

```diff
@@ -270,6 +271,34 @@ def sr_cleanup(self, force=False):
         cmd = 'hdbnsutil -sr_cleanup{}'.format(' --force' if force else '')
         self._run_hana_command(cmd)
 
+    def _parse_replication_output(self, output):
+        """
+        Utility function to parse output of
+        systemReplicationStatus.py
+        TODO: Parse table data
+        TODO: Parse local state
+        """
+        return {}
+
+    def get_replication_status(self):
+        """
+        Get system replication status (parsed output
+        of systemReplicationStatus.py).
+        """
+        cmd = 'HDBSettings.sh systemReplicationStatus.py'
+        result = self._run_hana_command(cmd, exception=False)
+        status = self._parse_replication_output(result.output)
+        rcmap = {
```

Comment:
Usually in Python I use strings since there isn’t much of a performance concern and it works even with older versions (enum is only available in newer versions of Python). But since we use it already I’ll do it here too :)